Add I need Help page (#796)
PrestaShop/docs: src/content/1.7/faq/i-need-help.md

---
title: I need help
---

# I need help

If you have questions or need help, please do not open a GitHub issue as we use GitHub issues only to discuss about bugs and new features for the PrestaShop project.

> The very best place to start looking for an answer is
> - here (the Developer documentation) if you are a developer

Reviewer: You'll need to add a blank space before a bulleted list
```suggestion
The very best place to start looking for an answer is:

- Here (the Developer documentation), if you are a developer
```

> - the [User's guide](https://doc.prestashop.com/display/PS17/User+Guide) if you are a user

Reviewer: ```suggestion
- The [User's guide](https://doc.prestashop.com/display/PS17/User+Guide) if you are a user
```

{{% notice tip %}}
If you are a developer and you think there is a missing item in the Developer documentation, consider [creating an issue on the Documentation Repository](https://github.com/PrestaShop/docs/issues). This will help us planning what to do to improve it.
{{% /notice %}}

> If you need help with your shop or third-party modules, please consider [the PrestaShop Company support plans](https://www.prestashop.com/support).

Reviewer: ```suggestion
If you need help with your shop or third-party modules, please have a look at the PrestaShop Company's [support plans](https://www.prestashop.com/support).
```

If you have questions about a PrestaShop module or theme you have downloaded or bought on the PrestaShop Addons Marketplace, you can contact its developers through the link provided on their Marketplace page.

> Finally, you can also ask for help from other PrestaShop users:

Reviewer: Again, blank line needed
```suggestion
Finally, you can also ask for help from other PrestaShop users:

```

- [on the Community chat](https://github.com/PrestaShop/open-source/blob/master/slack/readme.md) (Slack)
- [on the Community forums](https://www.prestashop.com/forums/)

> - [on Stack Overflow](https://stackoverflow.com/questions/tagged/prestashop)

Reviewer: Missing EOL
```suggestion
- [on Stack Overflow](https://stackoverflow.com/questions/tagged/prestashop)

```

---
title: I need help
---

# I need help

If you have questions or need help, please do not open a GitHub issue as we use GitHub issues only to discuss about bugs and new features for the PrestaShop project.

The very best place to start looking for an answer is:

- Here (the Developer documentation), if you are a developer
- The [User's guide](https://doc.prestashop.com/display/PS17/User+Guide) if you are a user

{{% notice tip %}}

> If you are a developer and you think there is a missing item in the Developer documentation, consider [creating an issue on the Documentation Repository](https://github.com/PrestaShop/docs/issues). This will help us planning what to do to improve it.

Reviewer: ```suggestion
If you are a developer and you think there is a missing item in the Developer documentation, consider [creating an issue on the Documentation Repository](https://github.com/PrestaShop/docs/issues). This will help us discuss what to do to improve it.
```
